apps/main/src/dex/components/PageRouterSwap/index.tsx

```diff
@@ -85,6 +86,9 @@ const QuickSwap = ({
   const stableMaxSlippage = useUserProfileStore((state) => state.maxSlippage.stable)
   const isStableswapRoute = routesAndOutput?.isStableswapRoute
   const storeMaxSlippage = isStableswapRoute ? stableMaxSlippage : cryptoMaxSlippage
+  const { isHighImpact } = routesAndOutput
+    ? getSlippageImpact(storeMaxSlippage, routesAndOutput)
+    : { isHighImpact: false }
```

Comment:
this looks like another case of showing incorrect data during loading or when errors occur. Maybe better to leave it undefined?
```suggestion
  const isHighImpact = routesAndOutput && getSlippageImpact(storeMaxSlippage, routesAndOutput).isHighImpact
```

Reply:
I don't think there's a visual difference between false and undefined. In both cases the message won't get shown